lib/robot.rb

```diff
@@ -14,6 +14,7 @@ def initialize(name)
         (name || ENV["NAME"]), [], nil
 
       redis_connection = Redis.new(:host => (ENV["REDIS_HOST"] || "127.0.0.1"),
+                                   :db   => (ENV["REDIS_DB"]   || 15),
```

Comment:
Good call, done. :)

On Thu, 27 Aug 2015 at 01:01 Taylor Blau <notifications@github.com> wrote:

> In lib/robot.rb
> <https://github.com/ttaylorr/slacker/pull/28#discussion_r37992266>:
>
> > @@ -14,6 +14,7 @@ def initialize(name)
> >          (name || ENV["NAME"]), [], nil
> >
> >        redis_connection = Redis.new(:host => (ENV["REDIS_HOST"] || "127.0.0.1"),
> > +                                   :db   => (ENV["REDIS_DB"]   || 15),
>
> Since DB 0 is the default in Redis when opening new connections with that
> parameter unspecified, how about we change this default to 0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ttaylorr/slacker/pull/28/files#r37992266>.
>